keyboards/handwired/split65/readme.md

```diff
@@ -0,0 +1,25 @@
+# Split65
+
+An split 65% keyboard with two OLEDs
+
+Keyboard Maintainer: [Percy-JW2](https://github.com/PercyJW-2)
+Hardware Supported: ProMicro and compatibles, Proton C
+Hardware Availability: [Repository](https://github.com/PercyJW-2/Split65)
+
+Make example for this keyboard with an ProMicro:
+
+    make handwired/split65/promicro:default
+    
+Make example for this keyboard with an Proton C:
```

Comment:
This can be done by adding `CTPC=yes` to the command line, too.

Reply:
I'm not really sure what you mean with that. What does this option do?

Reply:
`CTPC = yes` enables "convert to proton c", which remaps pin names and such to use the correct pins on the Proton C, as well as set up the config to use the STM32F303 MCU.  No need to have a separate folder for the proton C, technically.

Reply:
and this also works with split keyboards? Because this was the main reason I created the separate folder for the proton c

Reply:
Yes, it definitely does.  I have it set up with my corne layout (`/layouts/community/split_3x6_3/drashna`), in fact. 

Though, if you're using different serial modes, then it may be best to have separate revisions, then.